.gitignore

```diff
@@ -1,6 +1,5 @@
 .idea
 /web.config
-js/config-local.js
```

Comment:
I noticed this too late, but this should not have been ignored. This had my local configuration setup in it, and when I pulled down the branch, this file was blown away. :(